docs/deployment-gcp.md

```diff
@@ -0,0 +1,134 @@
+# Deploying Canopy on Google Cloud Platform (GCP) - Step-by-Step Guide
```

Comment:
There are multiple ways to deploy a service on GCP - I think the title or the introduction should say "GCP Cloud Run"